src/plugins/dashboard/public/dashboard_actions/library_notification_action.tsx

```diff
@@ -66,8 +62,6 @@ export class LibraryNotificationAction implements Action<LibraryNotificationActi
     );
   };
 
-  public readonly MenuItem = reactToUiComponent(this.LibraryNotification);
```

Comment:
Now that reactToUiComponent no longer loads this component in a separate react tree, can you remove `KibanaThemeProvider` wrapper in `MenuItem`?

Reply:
That is the goal—to be able to preserve the React context, in general. But in this specific case, I don't know, it needs to be tested case-by-case, as React context might have been broken in some other place as well.